### ISSUE TITLE
One question about how to draw the training loss plot.

### ISSUE THREAD

**Issue by 14H034160212** [pytorch/fairseq#3137]
Hi, I am running the RoBERTa-large and fine-tune it on the RACE dataset. I can see the training loss changing from the command window. But I want to draw the plot to record the change of the -training loss. Does anyone know how to do that? Thanks.
![image](https://user-images.githubusercontent.com/23516191/104810065-1d2fc100-5857-11eb-8c20-b8303ecd8c70.png)

**Comment by TarunTater** [comment]
bump
